package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "node-js-course",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "prestart": "babel src -d dist",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon ./dist/app.js"
```

Comment:
Use `./node_modules/.bin/nodemon ./dist/app.js`, please

FYI: If this case i need install nodemon globally, but if i don't wish install it globally. A case above provide us to avoid this issue.

Reply:
Done, thx
Well.. I don't have nodemon installed globally and this command worked for me. What's the difference ?